src/CoreShop/Component/Core/Order/Transformer/CartToSaleTransformer.php

```diff
@@ -68,7 +69,23 @@ public function transform(ProposalInterface $cart, ProposalInterface $sale)
                 $sale->setShippingTaxRate(0);
             }
 
-            $sale->setAdditionalData($cart->getAdditionalData());
+            $cartAdditionalData = $cart->getAdditionalData();
+            $saleAdditionalData = $sale->getAdditionalData();
```

Comment:
since the sale is actually new here, what if it returns null? than the data is never transfered

Reply:
That's not possible: 
![image](https://user-images.githubusercontent.com/700119/55427232-d6d87780-5586-11e9-8e14-0b3850e94fbb.png)

Reply:
actually it is: if the class is missing right? I think we should just double check to prevent issues on order creation.